cmd/registry-api-descriptor-template/main.go

```diff
@@ -44,7 +45,15 @@ func main() {
 
 	tmpl := template.Must(template.New(filename).Funcs(funcMap).ParseFiles(path))
 
-	if err := tmpl.Execute(os.Stdout, v2.APIDescriptor); err != nil {
+	data := struct {
+		RouteDescriptors []v2.RouteDescriptor
+		ErrorDescriptors []errcode.ErrorDescriptor
+	}{
+		RouteDescriptors: v2.APIDescriptor.RouteDescriptors,
+		ErrorDescriptors: errcode.GetErrorDescriptors(),
```

Comment:
I assumed, perhaps incorrectly, that we wanted all errors from all groups.  While in general its possible that we may want to pick a certain group, in this case I think all errors registered will all our one exe and therefore should be part of the list.  no?   

Or, let me turn it around, which errors would we want to exclude?  Even the one defined by errcode should be included, I would think, since it defines the generic one that 'error' maps to.